test/e2e/driver/mouse.py

```diff
@@ -42,3 +42,16 @@ def press_and_move(
     move(obj, x, y, dx, dy, step, sleep)
     squish.mouseRelease(mouse)
     time.sleep(1)
+
+def scroll(
+        obj,
+        x: int,
+        y: int,
+        dx: int,
+        dy: int,
+        steps: int = 1,
+        sleep: float = 0
+):
+    for _ in range(steps):
+        squish.mouseWheel(obj, x, y, dx, dy, squish.Qt.ControlModifier)
```

Comment:
Just wondering why the CTRL modifier here?

Reply:
Nice finding! I'll update to `NoModifier`

Reply:
Cool, it would actually zoom in some cases imho, instead of just plain scroll

Reply:
True. Was never supposed to use the ctrl modifier. Probably typo or copy paste error 😄 
Thanks!